f/fcs/fn_fcsBriefing.sqf

```diff
@@ -0,0 +1,34 @@
+// F3 - FCS Component Briefing Generator
+// Credits and documentation: https://github.com/folkarps/F3/wiki
+/*
+This function generates a briefing tab describing the operation of the F3 FCS. It is called from f_fnc_fcsInit, if the tab has not already been generated by a previous call.
+*/
+// ====================================================================================
+
+if (!hasInterface) exitWith {}; // Exit if not a player.
+
+_fcs = player createDiaryRecord ["diary", ["F3 Enhanced FCS","
```

Comment:
This is nice! My only concern is that this will appear at the _top_ of the Briefing tab, whereas it makes more sense at/near the bottom (e.g. with credits and administration). Could `f_fnc_fcsInit` be moved to before the briefing in `init.sqf`.

An alternative would be to add this to it's own subject with `createDiarySubject`.

Reply:
This is consistent with other "supplemental" items such as Mission Conditions, F3 Nametags, and the mapClickTeleport HALO instructions. I could move it, but I'm not all that bothered about it to be honest.

Reply:
In local testing the FCS briefing tab seems to appear at the bottom anyway.

Reply:
Hmm, so it does. Probably `spawn` for briefing vs `call` for this (and maybe also the `waitUntil` in briefing. Guess if it works it works